source/computer/commands.rst

```diff
@@ -63,54 +62,489 @@ Linux/macOS 下有成百上千个命令，每个命令都有众多选项。这
 
     -   其他
 
+        - `df`_
+        - `du`_
         - `ssh`_
         - `scp`_
         - `rsync`_
+        - `wget`_
+        - `top`_
 
 ----
 
 cat
 ---
 
+``cat`` 命令的命名来源于 con\ **cat**\ enate（即“拼接”）。该命令可以输出某个文件的
+内容，或将多个文件\ **拼接**\ 起来::
+
+    # 将一个文件的内容输出到终端
+    $ cat file
+    
+    # 同时显示行号
+    $ cat -n file
+    
+    # 将文件 file1 和 file2 合并到一个文件 target_file 中
+    $ cat file1 file2 > target_file
+
+diff
+----
+
+``diff`` 命令的命名来自 **diff**\ erence。该命令可以用来逐行比较文件的异同::
+
+    # 比较 /etc/passwd 和 /etc/passwd- 的异同
+    # 结果显示 /etc/passwd- 多了第 40 行
+    $ diff diff /etc/passwd /etc/passwd-
+    39a40
+    > guest:x:1001:1001::/home/guest:/bin/bash
+
+df
+--
+
+``df`` 命令的命名来自 **d**\ isk **f**\ ree（可使用的磁盘空间）。该命令可以获取硬盘被占用了多少空间，
+目前还剩下多少空间等信息。
+
+默认显示单位为 KB，使用 ``-h`` 或 ``--human-readable`` 选项可以用
+可读性较高的方式来显示信息::
+
+    $ df -h
```

Comment:
1. The default unit is not KB on macOS.
2. `-h` works on macOS, but `--human-readable` doesn't:
```suggestion
::

    $ df -h
```